cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -249,10 +252,10 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
 
         status_db: Store = self.status_db
         rna_case = status_db.get_case_by_internal_id(internal_id=case_id)
-        rna_dna_sample_case_map: Dict[str, Dict[str, list]] = self.create_rna_dna_sample_case_map(
-            rna_case=rna_case
-        )
-        for rna_sample_id in rna_dna_sample_case_map:
+        rna_dna_bundles: List[RnaDnaBundle] = self.create_rna_dna_bundles(rna_case=rna_case)
```

Comment:
I would omit the named argument here since it is a DRY violation and there is only a single argument, so naming it does not really matter. We can't mix it up with any other argument and there is no risk of using the wrong position for it

Reply:
But it does make the contract explicit and if you want to change the argument in the future you can refactor it via the editor which I am not aware of it is possible with positional args. I think we should not require named args for one arg functions, but I do not think we should discourage it.

Reply:
Would this not be a problem which only affects methods with multiple inputs? I think I am just having a hard time seeing what issue may arise when refactoring.